.readthedocs.yml

```diff
@@ -10,8 +10,8 @@ sphinx:
    configuration: docs/conf.py
 
 # Optionally build your docs in additional formats such as PDF
-formats:
-   - pdf
+# formats:
```

Comment:
Remove commented out lines

Reply:
It works locally in creating a directory with something that you supposedly can manually run to get PDFs, it doesn't go all the way to generate the final PDF locally. Readthedocs attempts to actually generate the final output on site regardless what the configuration is on site and run into error there.